Userland/Libraries/LibJS/Runtime/Temporal/Calendar.cpp

```diff
@@ -67,6 +68,216 @@ ReadonlySpan<StringView> available_calendars()
     return calendars.span();
 }
 
+// 12.2.2 CreateCalendarMethodsRecord ( calendar, methods ), https://tc39.es/proposal-temporal/#sec-temporal-createcalendarmethodsrecord
+ThrowCompletionOr<CalendarMethodsRecord> create_calendar_methods_record(VM& vm, Variant<String, NonnullGCPtr<Object>> const& calendar, Vector<CalendarMethod> const& methods)
```

Comment:
Seems like `calendar` can be passed by value and then `move`d into `receiver` below?

Reply:
Can `methods` be a `Span` (or `ReadonlySpan`) to avoid needing to create a vector?